CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v6.0.0-beta.5](https://github.com/nextcloud/nextcloud-vue/tree/v6.0.0-beta.5) (2022-08-30)
+
+[Full Changelog](https://github.com/nextcloud/nextcloud-vue/compare/v6.0.0-beta.4...v6.0.0-beta.5)
+
+### :boom: Breaking changes
+
+- Fix emptyContent once more [\#3106](https://github.com/nextcloud/nextcloud-vue/pull/3106) ([jotoeri](https://github.com/jotoeri))
+- Fix EmptyContent Reactivity [\#3090](https://github.com/nextcloud/nextcloud-vue/pull/3090) ([jotoeri](https://github.com/jotoeri))
+- Make unread counter primary colour [\#3070](https://github.com/nextcloud/nextcloud-vue/pull/3070) ([GretaD](https://github.com/GretaD))
```

Comment:
I believe this is not breaking https://github.com/nextcloud/nextcloud-vue/pull/3070#issuecomment-1231656980